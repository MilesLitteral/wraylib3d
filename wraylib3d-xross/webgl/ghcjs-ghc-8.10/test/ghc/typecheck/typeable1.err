user error (Pattern match failure in do expression at typeable1.hs:22:5-11)
