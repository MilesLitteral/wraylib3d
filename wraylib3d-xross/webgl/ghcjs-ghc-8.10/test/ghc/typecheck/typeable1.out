(ComposeK * * Maybe Maybe,Int)
(ComposeK * * Maybe,Maybe)
(ComposeK * *,Maybe)
